src/Apps/MyCollection/Routes/EditArtwork/Utils/categoryOptions.ts

```diff
@@ -0,0 +1,29 @@
+export const categoryOptions = [
```

Comment:
**reminder:** can you include this file in the list at the end of this notion doc https://www.notion.so/artsy/Adding-new-medium-types-to-Artsy-023c788ef7b1427296fba008d5ad2d74 once this gets merged

Reply:
Thanks for the reminder! I've added the link to the file 👍